src/core/handlers/HttpHandler.ts

```diff
@@ -102,7 +102,7 @@ export class HttpHandler extends RequestHandler<
     })
 
     devUtils.warn(
-      `Found a redundant usage of query parameters in the request handler URL for "${method} ${path}". Please match against a path instead and access query parameters in the response resolver function using "req.url.searchParams".`,
+      `Found a redundant usage of query parameters in the request handler URL for "${method} ${path}". Please match against a path instead and access query parameters by constructing a URL instance out of the request.url string. Please see the documentation for more details: https://mswjs.io/docs/recipes/query-parameters`,
```

Comment:
```suggestion
      `Found a redundant usage of query parameters in the request handler URL for "${method} ${path}". Please match against a path instead and access query parameters using "new URL(request.url).searchParams" instead. Learn more: https://mswjs.io/docs/recipes/query-parameters`,
```

What do you think if we rephrase it this way? Give a copy-pastable snippet and also shorter into for the docs?

Reply:
Definitely a better phrasing! Will update :)